gradio/interface.py

```diff
@@ -735,18 +736,35 @@ class TabbedInterface(Blocks):
     """
 
     def __init__(
-        self, interface_list: List[Interface], tab_names: Optional[List[str]] = None
+        self,
+        interface_list: List[Interface],
+        tab_names: Optional[List[str]] = None,
+        theme: str = "default",
+        analytics_enabled: Optional[bool] = None,
+        title: str = "Gradio",
+        css: Optional[str] = None,
     ):
         """
         Parameters:
             interface_list: a list of interfaces to be rendered in tabs.
             tab_names: a list of tab names. If None, the tab names will be "Tab 1", "Tab 2", etc.
+            theme: which theme to use - right now, only "default" is supported.
+            analytics_enabled: whether to allow basic telemetry. If None, will use GRADIO_ANALYTICS_ENABLED environment variable or default to True.
+            mode: a human-friendly name for the kind of Blocks interface being created.
```

Comment:
We should remove mode from the docstring since it's always "tabbed_interface"

Reply:
My bad, thanks